utils/unittest/epoch_builder.go

```diff
@@ -374,5 +381,48 @@ func (builder *EpochBuilder) addBlock(block *flow.Block) {
 		require.NoError(builder.t, err)
 	}
 
-	builder.blocks[block.ID()] = block
+	builder.blocksByID[block.ID()] = block
+	builder.blocks = append(builder.blocks, block)
+}
+
+// AddBlocksWithSeals for the n number of blocks specified this func
+// will add a seal for the second highest block in the state and a
+// receipt for the highest block in state to the given block before adding it to the state.
+// NOTE: This func should only be used after BuildEpoch to extend the commit phase
+func (builder *EpochBuilder) AddBlocksWithSeals(n int, counter uint64) *EpochBuilder {
+	for i := 0; i < n; i++ {
+		// given the last 3 blocks in state a <- b <- c
+		// the new block will contain
+		// - seal for c
+		// - execution receipt for b
+		// - execution result for a
+		a := builder.blocks[len(builder.blocks)-1]
+		b := builder.blocks[len(builder.blocks)-2]
+
+		// get execution result for third highest block
+		//builder.blocks[len(builder.blocks)-3].Payload.Receipts
+
+		// get receipt for the highest block
+		receiptF := ReceiptForBlockFixture(a)
+
+		// build a new block
+		// this new block contains a seal for previous highest block and a receipt for the highest block
+		block := BlockWithParentFixture(a.Header)
+		seal := Seal.Fixture(
+			Seal.WithResult(b.Payload.Results[0]),
+		)
+		block.SetPayload(flow.Payload{
+			Receipts: []*flow.ExecutionReceiptMeta{receiptF.Meta()},
+			Results:  []*flow.ExecutionResult{&receiptF.ExecutionResult},
+			Seals:    []*flow.Seal{seal},
```

Comment:
Can use the helper functions `WithReceipts` and `WithSeals` here: https://github.com/onflow/flow-go/blob/8f5f429ff969444c527e6a73b1474a37f6c7f18f/utils/unittest/fixtures.go#L259

Reply:
https://github.com/onflow/flow-go/pull/1940/commits/6e64634d3c34f3e167f62d74e1c1cf5e49f5d740

Reply:
https://github.com/onflow/flow-go/pull/1940/commits/c6a183c6f9e27cdf6e2e7625f129aaffa5932c16